value.go

```diff
@@ -14,11 +14,38 @@ var (
 	ErrTypeSyntax = errors.New("syntax error parsing type string")
 )
 
+var (
+	NullUint8    = &Value{Type: TypeUint8}
+	NullUint16   = &Value{Type: TypeUint16}
+	NullUint32   = &Value{Type: TypeUint32}
+	NullUint64   = &Value{Type: TypeUint64}
+	NullInt8     = &Value{Type: TypeInt8}
+	NullInt16    = &Value{Type: TypeInt16}
+	NullInt32    = &Value{Type: TypeInt32}
+	NullInt64    = &Value{Type: TypeInt64}
+	NullDuration = &Value{Type: TypeDuration}
+	NullTime     = &Value{Type: TypeTime}
+	NullFloat32  = &Value{Type: TypeFloat32}
+	NullFloat64  = &Value{Type: TypeFloat64}
+	NullBool     = &Value{Type: TypeBool}
+	NullBytes    = &Value{Type: TypeBytes}
+	NullString   = &Value{Type: TypeString}
+	NullIP       = &Value{Type: TypeIP}
+	NullNet      = &Value{Type: TypeNet}
+	NullTypeType = &Value{Type: TypeNet}
+	Null         = &Value{Type: TypeNull}
+)
+
 type Value struct {
 	Type  Type
 	Bytes zcode.Bytes
 }
 
+type Allocator interface {
```

Comment:
Nit: Put this before the Value declaration so the Value declaration stays with its constructor.